code/game/mecha/mecha.dm

```diff
@@ -1070,6 +1070,30 @@
 	return
 
 
+/obj/mecha/verb/checkSeat()

```

Comment:
There are certain items, that when you move them, cause other worn items to fall off, like the jumpsuit and your ID, belt, pockets, and PDA - or a hardsuit and an oxytank. You're already not able to drop items intentionally, but this is intended for items such as the above which would cause other things to fall if moved.

I am wondering what exactly I would do if someone removed one of the above items, that would cause others to fall loose.

GRANTED, I probably could add a check to an item whether a given piece of equipment would cause items to fall off or not when removed, and check that in its interaction procs, along with your `drop_item` system.